docs/modules/reference/pages/configuration/tuning-activemq.adoc

```diff
@@ -51,14 +51,14 @@ echo 'org.opennms.instance.id=MyNMS' > "$\{OPENNMS_HOME}/etc/opennms.properties.
 ----
 
 Update the Minion's instance ID accordingly to match the {page-component-title} instance.
-The same ID instance property should be present in `${MINION_HOME}/etc/custom.system.properties`.
+The same ID instance property should be present in `$\{MINION_HOME}/etc/custom.system.properties`.
```

Comment:
These changes have already been pushed to foundation-2021.  Will that conflict with this commit?

Reply:
@mmahacek. I'm not sure. I branched from the Release 28.x branch to work on this Jira, as the meta-data changes Christian did are a new feature. These issues showed up when I built locally, so wasn't sure how else to address them.